nrpy/tests/DOCTEST-gpu__cuda_utilities.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Please update to use `validate_strings()` (in nrpy/helpers/generic.py)

Reply:
All instances of using json have been replaced by `validate_strings()`